.github/workflows/windows.yml

```diff
@@ -0,0 +1,50 @@
+name: Build Windows release
+
+on:
+  push:
+    branches: [ "main", "ci-*"]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build-windows-release:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ['3.12']
+      fail-fast: true
+
+    steps:
+    - uses: actions/checkout@v4.1.2
+      with:
+        fetch-depth: 0
+
+    - name: Get short commit ref
+      run: |
+        version=$(
+            git describe --tags --dirty \
+            | sed -e 's/^v//' -e 's/\([^-]*-g\)/r\1/;s/-/./g'
+          )
+        echo "PAPIS_VERSION=$version" >> "$GITHUB_ENV"
+        
+
+    - name: Set version in pyproject.toml
+      run: |
+        set -x \
+        sed -i \
+          "s/^version =/version = \"${{env.PAPIS_VERSION}}\"/" \
+          pyproject.toml
+    
+    - name: Build Windows binary
+      run: |
+        docker run -v ${{github.workspace}}:/src \
+         -e PIP_GROUPS=windows,develop \
+         -e SPECFILE=scripts/windows/papis.spec \
+         ghcr.io/kiike/pyinstaller-windows:latest
```

Comment:
It is a forked third party image. I made this to better support pyproject.toml, since the original image uses requirements.txt. My aim is to provide a pull request there sometime.

Reply:
We can use `pip-tools` or `uv pip compile` to get a `requirements.txt` pretty easily. Would that work?

Reply:
Thanks to your mention of building on windows, I tried a bit more and I could get the build to work fine. No container or WINE build needed anymore.